include/macros.h

```diff
@@ -74,6 +74,8 @@
 #define CUR_FORM_EQUIP(button) (gSaveContext.equips.buttonItems[CUR_FORM][button])
 #define CHECK_QUEST_ITEM(item) (((void)0, gSaveContext.inventory.questItems) & gBitFlags[item])
 #define REMOVE_QUEST_ITEM(item) (gSaveContext.inventory.questItems = (((void)0, gSaveContext.inventory.questItems) & (-1 - gBitFlags[item])))
+#define CHECK_DUNGEON_ITEM(item) (gSaveContext.inventory.dungeonItems[(void)0, gSaveContext.mapIndex] & gBitFlags[item])
```

Comment:
This caught my curiosity.
OoT defines this macro as
```
#define CHECK_DUNGEON_ITEM(item, dungeonIndex) (gSaveContext.inventory.dungeonItems[dungeonIndex] & gBitFlags[item])
```
(requiring the `dungeonIndex` parameter explicitly).
I tried to change the OoT version if this macro to this one, but it didn't match. The specific file I had problems with was `z_map_exp`.
I think it would be better to use the OoT version of this macro for now, at least until we have the files that use this macro decomped so we can properly test it here (we know some macros changed between OoT and MM).